tests/test_srt.py

```diff
@@ -686,3 +686,21 @@ def test_can_parse_index_leading_zeroes(input_subs, zeroes):
 
     reparsed_subs = srt.parse(out)
     subs_eq(reparsed_subs, input_subs)
+
+
+
+@given(st.lists(subtitles()))
+def test_parse_file_with_missing_index_on_first_line(input_subs):  # cf. issue #51
+    out = ""
+    for sub in input_subs:
+        lines = sub.to_srt().split("\n")
+        first_block_without_index = [lines[1], lines[2], lines[3]]
```

Comment:
So you're constructing a sub above the existing one at index 0, copying the three lines, and then continuing. Why add a new subtitle? Is there something I'm missing about why the following isn't possible? :-)

```
srt = sub.to_srt()
srt = srt[srt.find("\n"):]
```

Reply:
Sure, that's another perfectly valid approach :)
I added a commit to switch to that version as you seem to prefer it.